upgrading/automated_upgrades.adoc

```diff
@@ -42,6 +42,11 @@ Ensure that you have met all
 xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[prerequisites]
 before proceeding with an upgrade. Failure to do so can result in a failed
 upgrade.
+
+If you are using GlusterFS, see
+xref:../upgrading/glusterfs_upgrades#upgrading-openshift-when-using-glusterfs[Upgrading
+{product-title} When Using GlusterFS Container Native Storage] before
```

Comment:
Do not use `GlusterFS Container Native Storage` to refer to GlusterFS. Please use the phrase `Containerized GlusterFS`.